--- gstreamer1-plugins-good.spec.orig	2014-04-08 14:23:13.000000000 +0100
+++ gstreamer1-plugins-good.spec	2014-04-08 15:15:28.000000000 +0100
@@ -1,3 +1,4 @@
+%if 0%{?fc20}
 %global         majorminor      1.0
 
 # Turn off extras package on RHEL.
@@ -9,7 +10,7 @@
 
 Name:           gstreamer1-plugins-good
 Version:        1.2.3
-Release:        2%{?dist}
+Release:        2%{?dist}.stbtester1
 Summary:        GStreamer plugins with good code and licensing
 
 License:        LGPLv2+
@@ -17,6 +18,9 @@
 Source0:        http://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-%{version}.tar.xz
 Patch3:         0001-v4l2src-Check-for-obj-pool-NULL.patch
 Patch4:         0001-docs-fix-mismatched-para-tags.patch
+Patch11:        0001-v4l2-normalise-control-names-in-the-same-way-as-v4l2-ctl.patch
+Patch12:        0002-v4l2src-fix-support-for-mpegts-streams.patch
+Patch13:        0003-v4l2src-only-care-about-getting-correct-width-height-if-in-caps.patch
 
 BuildRequires:  gstreamer1-devel >= %{version}
 BuildRequires:  gstreamer1-plugins-base-devel >= %{version}
@@ -88,6 +92,9 @@
 %setup -q -n gst-plugins-good-%{version}
 %patch3 -p1
 %patch4 -p1
+%patch11 -p1
+%patch12 -p1
+%patch13 -p1
 
 
 %build
@@ -207,6 +214,9 @@
 
 
 %changelog
+* Tue Apr 8 2014 David Rothlisberger <david@rothlis.net> - 1.2.3-2.stbtester1
+- v4l2src fixes for #725632 and #725860, required for Hauppauge HDPVR.
+
 * Thu Mar 13 2014 Wim Taymans <wtaymans@redhat.com> - 1.2.3-2
 - Rebuild for libvpx ABI break. See #1068664
 - fix doc build
@@ -304,3 +314,4 @@
 
 * Wed Jul 18 2012 Brian Pepple <bpepple@fedoraproject.org> - 0.11.92-1
 - Initial Fedora spec.
+%endif
